nixos/doc/manual/installation/installing-usb.section.md

```diff
@@ -1,31 +1,54 @@
-# Booting from a USB Drive {#sec-booting-from-usb}
+# Booting from a USB flash drive {#sec-booting-from-usb}
 
-For systems without CD drive, the NixOS live CD can be booted from a USB
-stick. You can use the `dd` utility to write the image:
-`dd if=path-to-image of=/dev/sdX`. Be careful about specifying the correct
-drive; you can use the `lsblk` command to get a list of block devices.
+The image has to be written verbatim to the USB flash drive for it to be
+bootable on UEFI and BIOS systems. Here are the recommended tools to do that.
 
-::: {.note}
-::: {.title}
-On macOS
+## Creating bootable USB flash drive with a graphical tool {#sec-booting-from-usb-graphical}
+
+Etcher is a popular and user-friendly tool. It works on Linux, Windows and macOS.
+
+Download it from [balena.io](https://www.balena.io/etcher/), start the program,
+select the downloaded NixOS ISO, then select the USB flash drive and flash it.
+
+::: {.warning}
+Etcher reports errors and usage statistics by default, which can be disabled in
+the settings. Some users might not like that.
```

Comment:
```suggestion
the settings.
```

Reply:
I thought it might be relevant to note that there are people criticizing Etcher because of that. Some call it "spyware", even tho it does not collect personal data.

That's why i also list an alternative, which is sadly less user-friendly, because it has many versions to choose from (4 per arch).

i remove the sentence because it's quite opinionated